rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Kindly ensure that `Getters and Setters for @book and @person instance variables are present`. See suggestion below:

```suggestion
  attr_accessor :date, :book, :person
```